release_notes.py

```diff
@@ -165,13 +169,23 @@ def load_data(self, data):
     def get_header(self):
         return "Scripts"
 
-    def added_release_notes(self, cnt):
+    def added_release_notes(self, cnt, demisto_version, first_release):
+        if first_release:
```

Comment:
I think something is wrong with the logic here

if `first_release` is false - you still want to compare version against `demisto_version`